tests/system/small/test_dataframe.py

```diff
@@ -1161,6 +1161,23 @@ def test_apply_series_scalar_callable(
     pandas.testing.assert_series_equal(bf_result, pd_result)
 
 
+def test_apply_from_read_gbq_function(dataset_id_permanent):
```

Comment:
This should go in tests/system/small/test_remote_function.py

Reply:
Done (Created a test in test_remote_function.py and removed the old one).